internal/pkg/export/export.go

```diff
@@ -57,3 +61,20 @@ func Export(sbomID string, opts *options.ExportOptions) error {
 
 	return nil
 }
+
+func cleanupEdges(document *sbom.Document) {
```

Comment:
Does this mean it deletes the edges? Maybe a quick godoc string to explain the intent, since it looks like we hoist the edges rather than clean them

Reply:
It removed edges of nodes that don't exist in the Document -> adding M2M relationship in storage returns edges of root nodes in other documents.

Reply:
That makes sense. I think adding it to the docs would be good.

Reply:
Added comment.